.ci/azure-pipelines-api-client.yml

```
@@ -0,0 +1,62 @@
+﻿parameters:
+  - name: LinuxImage
+    type: string
+    default: "ubuntu-latest"
+  - name: GeneratorVersion
+    type: string
+    default: "4.3.1"
```

```suggestion
    default: "5.0.0-beta2"
```
Currenly there are issues with typescript APIs with the generator on 4.x versions (see [here](https://github.com/OpenAPITools/openapi-generator/issues/7282)). This is fixed in the v5 beta so targeting the v5 beta should fix the issue.